examples/remarketing/add_customer_match_user_list.py

```diff
@@ -191,6 +192,7 @@ def _add_users_to_customer_match_user_list(
         "Offline user data job with resource name "
         f"'{offline_user_data_job_resource_name}' has finished."
     )
+    # [END add_customer_match_user_list]
```

Comment:
move after line 300

Reply:
This region will replace what is currently displayed for Python (last code snippet at the bottom of this [page](https://developers.google.com/google-ads/api/docs/migration/user-data-service.md#python).

I am happy to change this to what you suggest but prefer to double check that we are aligned.